src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/SourceGenerators/Diagnostics/RazorSourceGeneratorResources.resx

```diff
@@ -121,7 +121,7 @@
     <value>Invalid RazorLangVersion</value>
   </data>
   <data name="InvalidRazorLangMessage" xml:space="preserve">
-    <value>Invalid value '{0}'' for RazorLangVersion. Valid values include 'Latest' or a valid version in range 1.0 to 8.0.</value>
+    <value>Invalid value '{0}' for RazorLangVersion. Valid values include 'Latest' or a valid version in range 1.0 to 11.0.</value>
```

Comment:
TIL. And also I'm definitely going to forget to update this again next year. Can we make the `11.0` an argument, and pass in `RazorLangVersion.Preview` when creating the diagnostic?

Also should probably add "'Preview'" to the valid values mentioned

Reply:
I added a test to catch this next year, but making this an argument sounds even better, thanks.